pennylane_lightning/core/src/simulators/lightning_qubit/StateVectorLQubit.hpp

```diff
@@ -695,9 +695,12 @@ class StateVectorLQubit : public StateVectorBase<PrecisionT, Derived> {
         // **__**__ for stride 2
         // ****____ for stride 4
         const std::size_t k = branch ? 0 : 1;
+#if defined(_OPENMP)
+#pragma omp parallel for collapse(2) default(none) shared(arr, half_section_size, stride, k)
+#endif
         for (std::size_t idx = 0; idx < half_section_size; idx++) {
```

Comment:
Thanks @xiaohanzai . 
Could you try to fuse these two loops into one to see if the performance can be improved, especially for the `stride=1` case?